lib/util/git/index.ts

```diff
@@ -435,7 +435,7 @@ export async function syncGit(): Promise<void> {
       throw new ExternalHostError(err, 'git');
     }
     const durationMs = Math.round(Date.now() - cloneStart);
-    logger.debug({ durationMs }, 'git clone completed');
+    logger.debug(`Git clone completed in ${durationMs}`);
```

Comment:
```suggestion
    logger.debug({ durationMs }, 'git clone completed');
```

Revert this as I use it for metrics in the app